.changeset/beige-bags-tell.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Please add a changelog entry for this

Reply:
@LekoArts Since we don't change the exposed API we normally don't add a changeset. Do you have a suggestion for changesets here?

Reply:
Something like

```md
Refactor the internal generation of request URLs to use a shared helper from `@clerk/backend`
```